builders/build-linux-rust/Dockerfile

```diff
@@ -12,17 +12,26 @@ COPY --from=ghcr.io/nordsecurity/uniffi-generators:v0.25.0-8 /bin/uniffi-bindgen
 COPY --from=ghcr.io/nordsecurity/uniffi-generators:v0.25.0-8 /bin/uniffi-bindgen-go /bin
 COPY --from=ghcr.io/nordsecurity/uniffi-generators:v0.25.0-8 /bin/uniffi-bindgen-cpp /bin
 
+# Multilib is not present in an arm64 debian but is used by libtelio
+RUN set -eux; \
+    export DEBIAN_FRONTEND=noninteractive; \
+    apt-get update; \
+    if apt-cache show gcc-10-multilib > /dev/null 2>&1; then \
+        apt-get install -y gcc-10-multilib; \
+    else \
+        echo "Package gcc-10-multilib does not exist. Skipping installation."; \
+    fi; \
+    rm -rf /var/lib/apt/lists/*
+
```

Comment:
Why was the simple entry replaced with this?

Reply:
Since the gcc-10-multilib is not present on the debian for arm64 this code has to install only if it is present.